crates/biome_css_parser/tests/css_test_suite/ok/tailwind/variant-container-query.css

```
@@ -0,0 +1,17 @@
+@variant @xl {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)